frontend/components/HtmlReferenceContainer/components/YotpoWidget/index.jsx

```diff
@@ -0,0 +1,108 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import styles from './style';
+import getYotpo from '../../../../helpers/getYotpo';
+
+/**
+ * Mounts yotpo script.
+ */
+class YotpoScript extends Component {
+  static propTypes = {
+    config: PropTypes.shape().isRequired,
+    productId: PropTypes.string.isRequired,
+    product: PropTypes.shape(),
+  };
+
+  static defaultProps = {
+    product: null,
+  }
+
+  /**
+   * Constructs
+   * @param {Objects} props props
+   */
+  constructor(props) {
+    super(props);
+    this.yotpoMountTimeout = undefined;
+    this.yotpoRefreshTimeout = undefined;
+  }
+
+  /**
+   * Mounts Yotpo script.
+   */
+  componentDidMount() {
+    if (!window.yotpo) {
+      getYotpo(this.props.config.yotpoAppKey);
+      return;
+    }
+    window.yotpo.initialized = false;
+    window.yotpo.clean();
+    this.yotpoMountTimeout = setTimeout(() => {
+      window.yotpo.initWidgets();
+      this.yotpoMountTimeout = undefined;
+    }, 500);
+  }
+
+  /**
+   * Update for yotpo fetching for different variant
+   * @param {Object} prevProps prevProps
+   */
+  componentDidUpdate(prevProps) {
+    if (this.props.productId !== prevProps.productId) {
+      window.yotpo.initialized = false;
+      window.yotpo.clean();
+      // 1500 Additional ms to account for refresh of widget properties and fetching of new widget.
```

Comment:
I believe this comment doesn't reflect the reality any more. Why it's 2 seconds now? I think it's little too much given that it's very often that user can actually see no reviews for some time after reaching the PDP on a real device.

Reply:
Mounted component for base product remain upon switching variants as of PWA 6.3.0 without the use of componentDidUpdate. 
I tested functionality with PWA 6.3.1 and have removed componentDidUpdate function.